ArduCopter/Parameters.cpp

```diff
@@ -1000,7 +1000,7 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @Param: FLIGHT_OPTIONS
     // @DisplayName: Flight mode options
     // @Description: Flight mode specific options
-    // @Bitmask: 0:Disable thrust loss check, 1:Disable yaw imbalance warning, 2:Release gripper on thrust loss, 3:Require position for arming
+    // @Bitmask: 0:Disable thrust loss check, 1:Disable yaw imbalance warning, 2:Release gripper on thrust loss, 3:Require position for arming, 4:Enable automated docking clamp release
```

Comment:
I think we could simplify this by removing the FLIGHT_OPTIONS bit and just enable the feature if the user defines the SERVOx_FUNCTION.